modules/deploy/pages/deployment-option/cloud/serverless.adoc

```diff
@@ -0,0 +1,157 @@
+= Provision a Serverless Cluster
+:description: Learn how to create a Serverless Cloud cluster.
+:page-cloud: true
+:page-beta: true
+
+Redpanda Serverless is the easiest way to start event streaming in the cloud: You can start streaming in 30 seconds. 
+
+== What is Redpanda Serverless?
+
+Redpanda Serverless is a managed streaming service (Kafka API) that completely abstracts users from scaling and operational concerns, and you only pay for what you use. It provides data isolation and data residency:
+
+* Your data is hosted on Redpanda Cloud clusters shared among multiple tenants. Other tenants never access your data.
+* Your data is stored in your Redpanda Cloud account, in the region you select. Data never leaves your account.
```

Comment:
Please replace "Your data is stored in your Redpanda Cloud account, in the region you select. Data never leaves your account." with:

Your data are stored in the Redpanda Cloud account, in the region you select. Data never leave the selected region.